docs/dotnet/dotnet-aspnetcore-metrics.md

```diff
@@ -0,0 +1,206 @@
+# Semantic Conventions for ASP.NET Core metrics
+
+**Status**: [Experimental, Feature-freeze][DocumentStatus]
+
+This document defines semantic conventions for ASP.NET Core metrics.
+
+<!-- toc -->
```

Comment:
The build system automatically creates a TOC from H2's. Building one manually is a maintenance problem. Titles frequently change.